packages/allure-newman/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "newman-reporter-allure",
```

Comment:
allure-newman

Reply:
there is a naming convention for newman reporters https://github.com/postmanlabs/newman#creating-your-own-reporter